### DISCUSSION TITLE
Speedruns Leaderboard

### DISCUSSION THREAD

**Discussion by divVerent** [divVerent/aaaaxy#30] [Spoilers]
I am going to update this post with all known published runs and records (in each category one per person). I am only including runs I have actually watched or otherwise verified; for verification I can use a .dem file (can be created using -demo_record=filename.dem), or just watch the video. Having the end screen (last frame of the credits) available is a big plus, as it also tells the speedrun category and exact version of the game, and contains an indicator if cheats or a modified game were used.

## Any%

| Place | Name | Time | Link |
|-|-|-|-|
| 1 | [yxwqy](https://www.youtube.com/@yxwqy) | 0:11:22.633 | https://www.youtube.com/watch?v=1eaXeiKp-hw |
| 2 | [Γεώργιος Αθερίδης](https://www.youtube.com/@user-cj2zt3zu1t) | 0:11:41.800 | https://www.youtube.com/watch?v=UCbcXhyac7E |
| 3 | MarcOsiris | 0:12:29.333 | video deleted  |
| 4 | [totor](https://www.youtube.com/@2z2o) | 0:12:31.850 | |
| 4 | [totor](https://www.youtube.com/@2z2o) | 0:13:15.300 | https://www.youtube.com/watch?v=bhVHSexiMvw |
| 5 | divVerent | 0:14:05.500 | https://www.youtube.com/watch?v=Bn0o1hITR1A |
| 6 | [ERROR_27] | 0:17:13.483 | https://www.youtube.com/watch?v=mqB3B2ewR8Q |
| 7 | [2-16は](https://space.bilibili.com/1773563679) | 0:28:13.183 | video deleted  (on touchscreen!) |

## No Teleports

| Place | Name | Time | Link | Also is No Escape |
|-|-|-|-|-|
| 1 | MarcOsiris | 0:12:29.333 | video deleted  | yes |
| 2 | divVerent | 0:21:03.550 | https://www.youtube.com/watch?v=T-9N5nW8E7s | yes |

(Note: every No Escape run is also a valid No Teleports run)

## 100% + All Notes + No Escape

| Place | Name | Time | Link |
|-|-|-|-|
| 1 | [Γεώργιος Αθερίδης](https://www.youtube.com/@user-cj2zt3zu1t) | 0:58:49.833 | https://www.youtube.com/watch?v=HwRPZNrYsWo |

## 100% + All Notes + All Secrets

| Place | Name | Time | Link |
|-|-|-|-|
| 1 | [Γεώργιος Αθερίδης](https://www.youtube.com/@user-cj2zt3zu1t) | 1:20:05.450 | https://www.youtube.com/watch?v=HwRPZNrYsWo |

# Special Runs

The following runs do not follow the usual rules and may be considered cheating, and thus are listed in their own categories here.

## VVVVVV mode

The entire game played with gravity flipping instead of jumping. To use this mode, pass the option `-cheat_vvvvvv`.

| Place | Name | Time | Link |
|-|-|-|-|
| 1 | [Арсений Алфеев](https://www.youtube.com/@chell44444) | 0:05:34.750 | https://www.youtube.com/watch?v=ed31Hn_YD5A |
| 2 | [lnee lnee](https://www.youtube.com/@lnee) | 0:07:26.950 | https://www.youtube.com/watch?v=l1oSBWWN21I |

## TAS

A TAS (Tool-Assisted Speedrun) is a run that uses pre-recorded or manually edited inputs for maximum accuracy. Such runs tend to be full of frame-perfect tricks humans could not perform at this consistency.

The best TAS runs (tool assisted speedruns) are shown here.

Some hints and patches for how to TAS this game are here: https://gist.github.com/UnmatchedBracket/e66dd05f8f23cbe5e8d86b58ffbf1ae8

| Category | Name | Time | Link |
|-|-|-|-|
| Any% | [Unmatched Bracket](https://www.youtube.com/@unmatchedbracket) | 0:09:31.850 | https://www.youtube.com/watch?v=UoCz2GMNTmw |

**Comment by ghost** [comment]
I think I have the current world record in the "No Teleports" category. Here is a link to the video I made: [https://youtu.be/3ql3DSi3Onw](https://youtu.be/3ql3DSi3Onw). I did this run in 0:12:29:333. Is there anything else I should do, other than just waiting for it to be verified?
(Btw I was just messing around in the beginning lol)

**Reply by divVerent** [reply]
Thanks, verified and added. You used the I Like To Move It Move It platform to not need escape :)

Sorry, I must have missed the Google Alert for the video.

**Reply by ghost** [reply]
Thank you for adding it!

**Comment by Geno1024** [comment]
Rank 5 in Any% has been deleted their proof video.

**Comment by ghost** [comment]
No Teleports WR proof video has been deleted

**Reply by divVerent** [reply]
That's sad... that was a nice run. I'm going to remove the video link then.
